app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -50,11 +47,12 @@ const AnalysisAvailable = ({
   analysis,
 }: {
   experiment: getExperiment_experimentBySlug;
-  analysis: AnalysisData | undefined;
+  analysis: AnalysisData;
```

Comment:
This should never be rendered if analysis is falsey, so I thought it would be better to type it that way